### PR TITLE
Fix bus kms

### LINKED ISSUE #365: Bus sums in result files are not the same
- transit_kms.txt bus sum is 415 966 km	
- vehicle_kms_vdfs.txt "bus" sum 415 869 km
  - Should we have and "other" class?
- vehicle_kms_areas.txt "bus" sum 
  - model area 503 376 km		
  - HS15 area 423 153 km
  - This is all transit modes, not only bus

### PR DESCRIPTION
**Pull request by zptro**
Closes #365.

### CODE REVIEW

File: Scripts/assignment/emme_assignment.py [hunk 1]
```diff
@@ -154,6 +154,7 @@ def aggregate_results(self, resultdata):
         # Aggregate and print vehicle kms and link lengths
         vdfs = {param.roadclasses[linktype].volume_delay_func
             for linktype in param.roadclasses}
+        vdfs.add(0)
```
Comment (johpiip) [inline Scripts/assignment/emme_assignment.py:157]: This looks a bit funny to me. Is there some other way to add "links without car traffic (= without volume delay function)" than manually?

File: Scripts/assignment/assignment_period.py [hunk 2]
```diff
@@ -483,20 +483,21 @@ def _calc_background_traffic(self, include_trucks=False):
...
-            link[self.extra("bus")] = freq
-            if link.type // 100 in param.bus_lane_link_codes[self.name]:
-                # Bus lane
-                link[background_traffic] = 0
-            else:
-                link[background_traffic] = freq
-            if include_trucks:
-                for ass_class in heavy:
-                    link[background_traffic] += link[ass_class]
+            if link.type > 100: # If car or bus link
```
Comment (johpiip) [inline Scripts/assignment/assignment_period.py:486]: Will link types 0-100 now be considered as vdf=0?

**Review by johpiip — APPROVED** [review]

File: Scripts/assignment/assignment_period.py (hunk 2 above)
Comment (zptro) [reply Scripts/assignment/assignment_period.py:486]: No, not because of this code. This code forces attribute `@bus` to zero for rail links.

**Review by johpiip — APPROVED** [review]
Sorry, I had missed that this had to be re-reviewed! I approved this. Related to this, I was wondering last weekend about the naming convention for `0` class. If we want it to be other than `0`, I think `no_vdf` is better than `missing` or `other`. Especially `missing` implies that there should be a vdf for these links even if there really shouldn't.

**Pull request merged**